jenkins/common.groovy

```diff
@@ -397,14 +397,15 @@ def cacheDrupalContent(dockerContainer, envUsedCache) {
 
       for (int i=0; i<VAGOV_BUILDTYPES.size(); i++) {
         def envName = VAGOV_BUILDTYPES.get(i)
-
-        if (!envUsedCache[envName]) {
-          dockerContainer.inside(DOCKER_ARGS) {
-            sh "cd /application && node script/drupal-aws-cache.js --buildtype=${envName}"
+        if(envName != "vagovdev") {
```

Comment:
Could potentially combine with the `if` statement below to avoid introducing another level of nesting. Also I'm thinking a comment would be useful since it won't be obvious why we are skipping the cache for dev

Reply:
I was thinking that too, but then the `else` statement below would take effect and I'm not sure we want that to happen for `vagovdev` dev as well

Reply:
Good catch about the comment, I'll add a comment after this is merged so we dont have to rerun the pipeline again.